dse-k8s-operator/README.md

```diff
@@ -89,58 +87,58 @@ watch the progress by getting the list of pods for the namespace, as
-racks, with a total of one node per rack.
+racks, with one node per rack.
 
 ## Cluster and Datacenter
 
-A DSE logical datacenter is the primary resource managed by the
-dse-k8s-operator. Within a single Kubernetes namespace:
+A logical datacenter is the primary resource managed by the
+cass-operator. Within a single Kubernetes namespace:
 
-- A single `DseDatacenter` resource defines a single-datacenter DSE cluster.
-- Two or more `DseDatacenter` resources with different `dseClusterName`'s define
-  separate and unrelated single-datacenter DSE clusters. Note the operator
+- A single `CassandraDatacenter` resource defines a single-datacenter cluster.
+- Two or more `CassandraDatacenter` resources with different `clusterName`'s define
+  separate and unrelated single-datacenter clusters. Note the operator
   manages both clusters since they reside within the same Kubernetes namespace.
-- Two or more `DseDatacenter` resources that share the same `dseClusterName`
-  define a multi-datacenter DSE cluster. The operator will join the DSE
-  instances in each datacenter into a logical topology that acts as a single DSE
+- Two or more `CassandraDatacenter` resources that have the same `clusterName`
+  define a multi-datacenter cluster. The operator will join the
+  instances in each datacenter into a logical topology that acts as a single
   cluster.
 
 For this guide, we define a single-datacenter cluster. The cluster is named
 `cluster1` with the datacenter named `dc1`.
 
 ## Racks
 
-DSE is rack-aware, and the `racks` parameter will configure the DSE operator to
+Cassandra is rack-aware, and the `racks` parameter will configure the operator to
 set up pods in a rack aware way. Note the Kubernetes worker nodes must have
 labels matching `failure-domain.beta.kubernetes.io/zone`. Racks must have
 identifiers. In this guide we will use `r1`, `r2`, and `r3`.
 
-## DSE Node Count
+## Node Count
 
-The `size` parameter is the number of DSE instances to run in the datacenter.
-For optimal performance, it's recommended to run only one DSE instance per
-Kubernetes worker node. The operator will enforce that limit, and DSE
+The `size` parameter is the number of nodes to run in the datacenter.
+For optimal performance, it's recommended to run only one server instance per
+Kubernetes worker node. The operator will enforce that limit, and
 pods may get stuck in the `Pending` status if there are insufficient Kubernetes
 workers available.
 
-We'll assume you have at least three worker nodes available, if you're working
-on a minikube or other setup with a single Kubernetes worker node, you must
-reduce the `size` value accordingly or set the `allowMultipleNodesPerWorker`
+We'll assume you have at least three worker nodes available. If you're working
+locally with minikube or another setup with a single Kubernetes worker node, you must
+reduce the `size` value accordingly, or set the `allowMultipleNodesPerWorker`
```

Comment:
Is `allowMultipleNodesPerWorker` in the root level of the datacenter spec?

Reply:
yup